src/module.ts

```diff
@@ -262,6 +262,11 @@ export default defineNuxtModule<ModuleOptions>({
           route: `/api/${options.base}/query`,
           handler: resolveRuntimeModule('./server/api/query')
         },
+        {
+          method: 'get',
+          route: `/api/${options.base}/find_one/:qid/**:slug`,
```

Comment:
Could you confirm it works with slug being without any `/` @farnabaz ?

Trying locally on Studio and seems that it sends back to me a 405

Reply:
Might be worth adding tests into radix3 https://github.com/unjs/radix3/blob/main/tests/insert.test.ts

Reply:
https://github.com/unjs/radix3/issues/8

Reply:
@Atinux I'm going to remove this endpoint and integrate Index with `/query` endpoint